duckduckgo/duckduckgo-privacy-extension
# Update popup and settings copy

**Reviewer:** @jdorweiler, @alohaas 



## Description:
Following up from #462 with further copy changes


## Steps to test this PR:
- build and load extension in FF / Chrome
- verify the following copy is in place:

#### Toggle animation states
   - _Added to Unprotected Sites_
   - _Removed from Unprotected Sites_


#### Hamburger Menu
- _Manage unprotected sites and other options_

#### Settings
- Description: _DuckDuckGo protects your privacy online with private search, tracker blocking, and site encryption._
 - List Heading: _Unprotected Sites_
  - List description: _These sites will not be enhanced by Privacy Protection._
 - Empty state: _No Unprotected Sites Yet_
  - Form reveal link: _Add Unprotected Site_
  - Form Button: _Add_


#### Feedback Confirmation
 - _To fix the issue for the time being, you can disable "Privacy Protection" to add it to your list of unprotected sites._

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
